content/en/_index.md

```diff
@@ -782,6 +949,9 @@ spanner_client.close
 
 $dlp = new DlpServiceClient();
 {{< /tab >}}
+{{< tab header="Terraform" >}}
+Clients run `terraform init`.
```

Comment:
Throw into a comment line

Reply:
clarify to "Clients run `terraform init` to initialize Terraform in the current working directory."

Reply:
removed